packages/react-ui-kit/package.json

```diff
@@ -25,9 +25,11 @@
     "jest": "25.1.0",
     "jest-emotion": "10.0.32",
     "react": "16.13.0",
+    "react-docgen-typescript": "^1.16.3",
     "react-dom": "16.13.1",
     "react-helmet": "5.2.1",
     "react-hot-loader": "4.12.20",
+    "react-styleguidist": "^11.0.5",
```

Comment:
pin versions

Reply:
nice catch, done in https://github.com/wireapp/wire-web-packages/pull/2937/commits/270e8255e4cdf5661fe50e01f755dabd0332b62a